### DISCUSSION TITLE
Welcome to Seeed_Arduino_K1100 Discussions!

### DISCUSSION THREAD

**Discussion by tianrking** [Seeed-Studio/SenseCraft-Wio#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GrayHatGuy** [comment]
Sensor development IOTs for LoRA/Helium. HW/SW/EMBEDDED

**Comment by Dangerousfish** [comment]
Loving the updates.

I'm currently using the S2104 and hoping to use the Wio to capture data sent remotely using the Wio-E5.

**Comment by meilily** [comment]
Hey community, we've just added a Wehook plug-in to this repository, so everything updated here on GitHub will sync with our community on Discord.
